app/(main)/recover-password/page.test.tsx

```diff
@@ -0,0 +1,120 @@
+import { recoverPassword } from '@/api/apiFunctions';
+import {
+  act,
+  fireEvent,
+  render,
+  screen,
+  waitFor,
+} from '@testing-library/react';
+import RecoverPassword from './page';
+
+// mock the recoverPassword function
+jest.mock('@/api/apiFunctions', () => ({
+  recoverPassword: jest.fn(),
+}));
+
+const mockPush = jest.fn();
+const getUser = jest.fn();
+
+let continueButton: HTMLElement, emailInput: HTMLInputElement;
+
+const mockUseAuthContext = {
+  getUser,
+  isSignedIn: false,
+};
+
+jest.mock('next/navigation', () => ({
+  useRouter() {
+    return {
+      push: mockPush,
+    };
+  },
+}));
+
+jest.mock('../../../context/AuthContextProvider', () => ({
+  useAuthContext() {
+    return {
+      ...mockUseAuthContext,
+    };
+  },
+}));
+
+jest.mock('react-hot-toast', () => ({
+  custom: jest.fn(),
+}));
+
+describe('Recover Password', () => {
+  beforeEach(() => {
+    jest.clearAllMocks();
+  });
+
+  test('should render the recover password page', () => {
```

Comment:
Other test files use `it`, is there a reason you're using `test` instead?

Reply:
it has been inconsistent across multiple test files.  @shashilo thoughts?